src/algoliasearch.helper.js

```diff
@@ -118,7 +118,7 @@ var version = require('./version');
  * just an object containing the properties you need from it.
  */
 function AlgoliaSearchHelper(client, index, options) {
-  if (client.addAlgoliaAgent && !doesClientAgentContainsHelper(client)) {
+  if (client.addAlgoliaAgent) {
```

Comment:
We should use `typeof ...` otherwise we might throw an error if the value is not a function.